verse/analysis/verifier.py

```diff
@@ -254,6 +422,29 @@ def compute_full_reachtube_step(
                     )
                     if config.incremental:
                         cache_tube_updates.extend(cache_tube_update)
+                elif consts.reachability_method == ReachabilityMethod.STAR_SETS:
```

Comment:
Within this elif, user should be able to specific which version of verify to use (rectangles or PCA stars)